### DISCUSSION TITLE
Icon fieldtype with external directory using S3 filesystem?

### DISCUSSION THREAD

**Discussion by JorisOrangeStudio** [statamic/cms#9822] [Q&A] [answered]
Is it possible to select a directory/folder from a external filesystem (DO Spaces or S3) inside the Icon fieldtype. We don't want to change it to an asset fieldtype, since the icon fieldtype has a more minimal interface for selecting icons.

**Comment by jasonvarga** [comment, marked as answer]
At the moment the icon fieldtype can only read from local directories.
